tests/execute/result/basic.sh

```diff
@@ -38,7 +38,7 @@ rlJournalStart
         run   "errr"   "/test/error"          ""         2
         run   "pass"   "/test/xfail-fail"     "fail"     0
         run   "fail"   "/test/xfail-pass"     "pass"     1
```

Comment:
Are we ok with this? I see no reason to add note when original and actual results are the same, no?